src/mappings/nabla/handleEvent.ts

```diff
@@ -0,0 +1,67 @@
+import { EventHandlerContext } from '../../processor'
+import * as backstopPoolAbi from '../../abi/backstop'
+import * as swapPoolAbi from '../../abi/swap'
+import * as routerAbi from '../../abi/router'
+import {
+    ZERO_ADDRESS,
+    getBackstopPool,
+    getRouter,
+    getSwapPool,
+} from './creation'
+import { createBackstopPool, createRouter, createSwapPool } from './creation'
+import { handleBackstopPoolEvent } from './backstopPoolEventHandler'
+import { handleRouterEvent } from './routerEventHandler'
+import { handleSwapPoolEvent } from './swapPoolEventHandler'
+import { hexToSs58 } from './addresses'
+
+export async function handleContractInstantiated(ctx: EventHandlerContext) {
+    const contractHexAddress = ctx.event.args.contract
+    if (contractHexAddress === ZERO_ADDRESS) {
```

Comment:
It probably never happens. I added this to be defensive. If for strange reason this would be the zero address, then the following code would fail (add an entry to the database with id 0).